main/templates/allauth/account/signup.html

```diff
@@ -1,11 +1,12 @@
 {% extends 'main/base.html' %}
```

Comment:
customised the register.html made eariler to match allauth template in terms of fileds, id, etc.